app/services/data/get-staging-court-reports.js

```diff
@@ -0,0 +1,74 @@
+const knex = require('../../../knex').stagingSchema
+const Promise = require('bluebird').Promise
+const OmCourtReports = require('wmt-probation-rules').OmCourtReports
```

Comment:
Is there going to be a function which reads the entire table from the original `staging.court_reports` as part of the `process-import`? Should it be called `get-staging-court-reports`? Maybe a rename here

Reply:
There will be a service that will read the entire table and get all the entries we need for the court-reporters table.  Think that service will be called get-staging-court-reporters and have a similar insert-court-reporters

Reply:
Should the name of the file not reference the table that it is querying?
ie `get-staging-court-reports` gets data from `staging.court_reports`
    `get-staging-court-reporters` gets data from `staging.court_reporters`

Reply:
Yep, sounds good. Lets get that name changed.

Reply:
Updated